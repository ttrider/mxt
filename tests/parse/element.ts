describe("parse:element", () => {
test("a1", () => { expect("a1").toBe("a1"); expect("a1").toMatchSnapshot(); });
test("a2", () => { expect("a2").toBe("a2"); expect("a2").toMatchSnapshot(); });
test("a3", () => { expect("a3").toBe("a3"); expect("a3").toMatchSnapshot(); });
});