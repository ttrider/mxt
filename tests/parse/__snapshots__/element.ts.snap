// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parse:element a1 1`] = `"a1"`;

exports[`parse:element a2 1`] = `"a2"`;

exports[`parse:element a3 1`] = `"a3"`;
